Tensile/Source/ReferenceCPU.h

```diff
@@ -22,6 +22,9 @@
 
 #ifndef REFERENCE_CPU_H
 #define REFERENCE_CPU_H
+#if defined(max)
+#undef max
+#endif
```

Comment:
This is interesting - where is the macro for max coming from ?

Reply:
I'll delete and see if current windows includes don't complate, was likely an old battle with NOMINMAX.